src/commons/mocks/AssessmentMocks.ts

```diff
@@ -296,17 +296,17 @@ const mockSoundLibrary: Library = {
 export const mockRuneLibrary: Library = {
   chapter: Chapter.SOURCE_1,
   external: {
-    name: ExternalLibraryName.RUNES,
-    symbols: externalLibraries.get(ExternalLibraryName.RUNES)!
+    name: ExternalLibraryName.NONE,
+    symbols: externalLibraries.get(ExternalLibraryName.NONE)!
   },
   globals: mockGlobals
 };
 
 const mockBinaryTreeLibrary: Library = {
   chapter: Chapter.SOURCE_4,
   external: {
-    name: ExternalLibraryName.BINARYTREES,
-    symbols: externalLibraries.get(ExternalLibraryName.BINARYTREES)!
+    name: ExternalLibraryName.NONE,
+    symbols: externalLibraries.get(ExternalLibraryName.NONE)!
   },
   globals: mockGlobals
```

Comment:
Would this result in redundant test cases? Also, the names of the objects can be changed accordingly.

Reply:
yes these will become redundant test cases, im going to remove them